ra/ra.go

```diff
@@ -1152,12 +1160,32 @@ func (ra *RegistrationAuthorityImpl) validateFinalizeRequest(
 			ID:            authz.ID,
 			ChallengeType: solvedByChallengeType,
 		}
+
+		// Count the number of authorizations where the original CAA check was
+		// reused and the number where it was rechecked.
+		staleCAA, err := validatedBefore(authz, ra.clk.Now().Add(caaRecheckDuration))
+		if err != nil {
+			// This should never happen because we just checked the same thing
+			// inside ra.checkOrderAuthorizations.
+			return nil, err
+		}
+		if staleCAA {
+			caaRechecked++
+		} else {
+			caaReused++
+		}
+
 	}
 	logEvent.Authorizations = logEventAuthzs
 
 	// Mark that we verified the CN and SANs
 	logEvent.VerifiedFields = []string{"subject.commonName", "subjectAltName"}
 
+	// Log the number of CAA checks that were reused and the number that were
+	// rechecked.
+	ra.log.Info(fmt.Sprintf("Id %d finalizing order with %d Authzs (%d CAA reused, %d CAA rechecked)",
+		req.Order.RegistrationID, len(authzs), caaReused, caaRechecked))
```

Comment:
If you feel like tracking one fewer variables, this could be
```suggestion
		req.Order.RegistrationID, len(authzs), caaReused, len(authzs)-caaReused))
```

It would save one variable and a couple lines of code, but it might be less readable. Totally up to your preference.